scripts/globals/items/lucid_elixir_ii.lua

```diff
@@ -0,0 +1,20 @@
+-----------------------------------------
+-- ID: 5831
+-- Item: Lucid Elixir II
+-- Item Effect: Restores 75% of HP and MP
+-----------------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/msg")
+
+function onItemCheck(target)
+    if target:getMaxHP() == target:getHP() and target:getMaxMP() == target:getMP() then
+        return tpz.msg.basic.ITEM_UNABLE_TO_USE_2
+    end
+    return 0
+end
+
+function onItemUse(target)
+    target:addHP(target:getMaxHP()*0.75*ITEM_POWER)
```

Comment:
this is one of those case where "extra" parentheses can make the order of operations clearer.